webpack/assets/javascripts/bundle.js

```diff
@@ -9,5 +9,9 @@ require('./bundle_select2');
 require('./bundle_datatables');
 window.tfm = {
   tools: require('./foreman_tools'),
-  initByteSpinner: require('./jquery.ui.custom_spinners').initByteSpinner
+  numFields: {
```

Comment:
no need to import each function separately, you can just use `numFields: require('./jquery.ui.custom_spinners')` and all exports will be imported (like we do for tools)